src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/Baselines/net462/ConfigurationBinder/Version1/Bind_Instance.generated.txt

```diff
@@ -54,7 +54,7 @@ namespace Microsoft.Extensions.Configuration.Binder.SourceGeneration
         #endregion IConfiguration extensions.
 
         #region Core binding extensions.
-        private readonly static Lazy<HashSet<string>> s_configKeys_Program__MyClass = new(() => new HashSet<string>(StringComparer.OrdinalIgnoreCase) { "MyString", "MyInt", "MyList", "MyDictionary", "MyComplexDictionary" });
+        private readonly static Lazy<global::System.Collections.Generic.HashSet<string>> s_configKeys_Program__MyClass = new(() => new global::System.Collections.Generic.HashSet<string>(StringComparer.OrdinalIgnoreCase) { "MyString", "MyInt", "MyList", "MyDictionary", "MyComplexDictionary" });
```

Comment:
I do notice a lot of cases we still have the using statement for these (see line 29) keeping it doesn't hurt, but is there anything to clean up?

Reply:
Yes, this is included here: https://github.com/dotnet/runtime/blob/11702c4619a9086bbf50b22af56b6777351bd6b0/src/libraries/Microsoft.Extensions.Configuration.Binder/gen/Specs/BindingHelperInfo.cs#L170

I didn’t modify that part because I’m not sure what potential breakages it might cause. Removing it seems riskier, especially if any generated code references a collection type without using a fully qualified name. We are not breaking anything here as this part existed before.

Reply:
> it might not be needed to use fully qualified names for collections since they are only used in the case of binding to a custom type

This is not true. Collections used even without custom type. For example, `var dict = configuration.Get<IDictionary<string, string>>()`, this is using `IDictionary<,>` which is not custom type and the generated code will use the concrete `Dictionary<,>` too.